sky/skylet/job_lib.py

```diff
@@ -916,7 +915,7 @@ def _create_ray_job_submission_client():
 
 
 def _make_ray_job_id(sky_job_id: int) -> str:
-    return f'{sky_job_id}-{getpass.getuser()}'
+    return f'{sky_job_id}-{common_utils.get_current_user_name()}'
```

Comment:
I think we should avoid this one. In a remote API server with SSO, the user name could be an email address. Not sure skypilot/ray will be happy with `@` and `.` in the job id here.